compaction/task_manager_module.hh

```diff
@@ -396,6 +404,10 @@ public:
         , _exclude_current_version(exclude_current_version)
     {}
 
+    virtual std::string type() const override {
+        return "upgrade " + sstables_compaction_task_impl::type();
```

Comment:
table upgrade?

I'm concerned it might be confusing if all these similar task variants show the same type...